vcd/resource_vcd_external_network_v2.go

```diff
@@ -0,0 +1,453 @@
+package vcd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+var networkV2IpScope = &schema.Resource{
+	Schema: map[string]*schema.Schema{
+		"gateway": &schema.Schema{
+			Type:         schema.TypeString,
+			Required:     true,
+			Description:  "Gateway of the network",
+			ValidateFunc: validation.IsIPAddress,
+		},
+		"prefix_length": &schema.Schema{
```

Comment:
is this how named in UI?

Reply:
In UI you have a single field where prefix_length goes after `/`. Up for a discussion if we want to enter `gateway` field in such format although I always find it confusing when gateway is entered in CIDR mask (which looks to be UI sugar)
![image](https://user-images.githubusercontent.com/15804230/94525304-91107400-023c-11eb-8e09-9f17dc54115b.png)